ide/nativeimage.api/src/org/netbeans/modules/nativeimage/spi/debug/NIDebuggerProvider.java

```diff
@@ -194,4 +194,13 @@
     default Map<SourceInfo, List<Symbol>> listVariables(String name, boolean includeNondebug, int maxResults) {
         return null;
     }
+
+    /**
+     * Provide a list of source paths in the debuggee.
+     *
+     * @since 0.7
+     */
+    default List<SourceInfo> listSources(String nameRegexp) {
```

Comment:
A note in API changes would be good. Specify that the method can return `null`. State the `List` is immutable.

Reply:
The API changes were abandoned for now.